openapi/p4pa-pu-bff.openapi.yaml

```diff
@@ -74,6 +74,135 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/ServiceStatus'
+  /taxonomy/getOrganizationTypes:
+    get:
+      tags:
+        - Taxonomy
+      summary: "Retrieve distinct organization types for FE"
+      operationId: getOrganizationTypes
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
```

Comment:
define directly the array here as done in:
https://github.com/pagopa/p4pa-auth/blob/develop/openapi/p4pa-auth.openapi.yaml#L184